experiments/calibration/make_training_locations.jl

```diff
@@ -15,11 +16,14 @@ function diagnostics_lat_lon(nelements)
     depth = 0.1 # These don't matter
 
     domain = ClimaLand.Domains.SphericalShell(; radius, depth, nelements)
+
     num_long, num_lat, _ =
         ClimaLand.Diagnostics.default_diagnostic_num_points(domain)
     longs = collect(range(-180.0, 180.0, length = num_long))
     lats = collect(range(-90.0, 90.0, length = num_lat))
-    return lats, longs
+
+    mask = ClimaLand.landsea_mask(domain; threshold = 0.99)
```

Comment:
can update the doc string to note that this also returns the ClimaLand mask as a Field, while lat and lon are arrays.

this function is a little confusing because it does two unrelated things; they just happen to both need the `domain`, so it was convenient to package them in one function.

I think it would be clearer to make the change I suggest below, and get rid of this function.